WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -273,6 +273,12 @@ def execute( self, arguments ):
     os.environ['DIRACSITE'] = DIRAC.siteName()
     self.log.verbose( 'DIRACSITE = %s' % ( DIRAC.siteName() ) )
 
+    os.environ['DIRAC_CORES'] = str( self.ceArgs.get( 'Cores', 1 ) )
+    self.log.verbose( 'DIRAC_CORES = %s' % ( self.ceArgs.get( 'Cores', 1 ) ) )
+
+    os.environ['DIRAC_WHOLENODE'] = str( self.ceArgs.get( 'WholeNode', False ) )
+    self.log.verbose( 'DIRAC_WHOLENODE = %s' % ( self.ceArgs.get( 'WholeNode', False ) ) )
+
```

Comment:
I would say that, on a general note, using environment variables presents a degree of risk, so I would rather avoid using them.